middleware.js

```diff
@@ -198,12 +197,14 @@ module.exports = function(options) {
 
       mkdirp(dirname(cssPath), '0700', function(err) {
         if (err) {
-          return error(err);
+          error(err);
+          cssDone = true;
+          return doneWriting();
```

Comment:
as it was, in case of errors here, `next()` would never be called and the request would just hang there. We need to call doneWriting.